benchmark-service/benchmark/benchmark.py

```diff
@@ -200,8 +207,37 @@ async def compare(request, userdata):  # pylint: disable=unused-argument
     return await render_template('benchmark', request, userdata, 'compare.html', context)
 
 
+async def query_github(app):
+    global START_POINT
+    github_client = app['github_client']
+    request_string = f'/repos/hail-is/hail/commits?since={START_POINT}'
+
+    data = await github_client.getitem(request_string)
+    new_commits = []
+    for commit in data:
+        sha = commit.get('sha')
+        new_commits.append(commit)
+        log.info(f'commit {sha}')
+        START_POINT = commit.get('commit').get('author').get('date')
+        log.info(f'start point is now {START_POINT}')
+    log.info('got new commits')
+
+
+async def github_polling_loop(app):
+    while True:
+        await query_github(app)
+        log.info('successfully queried github')
+        await asyncio.sleep(60)
```

Comment:
Let's change this to 600 for now. That's 10 minutes. I don't want to blow out our API request limits to Github.

We'll merge this in and then we can reevaluate later once we've seen the logs in the production instance of the service.